pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -309,14 +309,14 @@ func TestReconcileProvisionOK(t *testing.T) {
 			AssertThatNamespace(t, username+"-dev", r.client).
 				HasNoOwnerReference().
 				HasLabel("toolchain.dev.openshift.com/owner", username).
-				HasLabel("toolchain.dev.openshift.com/type", "dev")
+				HasLabel("toolchain.dev.openshift.com/type", "dev").
+				HasNoLabel("toolchain.dev.openshift.com/revision")
```

Comment:
we could also verify the non-existence of the tier label
```suggestion
				HasNoLabel("toolchain.dev.openshift.com/revision").
				HasNoLabel("toolchain.dev.openshift.com/tier")
```